contrib/deploy-onefuzz-via-azure-devops/deploy-onefuzz.yml

```diff
@@ -121,6 +121,6 @@ stages:
           targetType: 'inline'
           script: |
             set -ex
-            ./onefuzz-cli-$(version).exe config --endpoint $(ONEFUZZ_SERVICE_URL) --client_id "$(AZURE_CLIENT_ID)" --client_secret "$(AZURE_CLIENT_SECRET)"
+            ./onefuzz-cli-$(version).exe config --endpoint $(ONEFUZZ_SERVICE_URL) --client_id "$(AZURE_CLIENT_ID)" --client_secret "$(AZURE_CLIENT_SECRET)" --authority "https://login.microsoftonline.com/$(AZURE_TENANT_ID)"
```

Comment:
This is a breaking change for anyone that has deployed this.

Can you make this optional?

Reply:
Hi Brian, we can absolutely make an update that makes this optional -- but if I might ask -- how is this breaking? 
The yml file has always asked for the AZURE_TENANT_ID, so how is this not semi-redundant for most users?

Reply:
If the concern is about hardcoding login.microsoftonline.com, it's not a breaking change since that is the default value used by the CLI if none is provided. I also don't understand why this is a breaking change either.

Reply:
@nharper285 I missed that the script already had the tenant id.

Reply:
Thanks Brian!